res/linuxPackaging.json

```diff
@@ -18,8 +18,8 @@
     "x-scheme-handler/ftp"
   ],
   "requires": [
-    "GConf2",
-    "libXScrnSaver",
+    "(GConf2 or gconf2)",
+    "(libXScrnSaver or libXss1)",
```

Comment:
What effect does this change have on leap though? Would the build's output RPM even continue to work with the new keyword on an older system?

Having *one* solution that works for more systems is better than rolling out two solutions that work in different ways. Hence why I am proposing in the main issue having a second build just for OpenSUSE. I am waiting on input from the team as to what would need to be done and allowed before putting in the work on something that may get rejected since it isn't setup how they'd prefer it get hosted.